### PR TITLE
Improve OpenStack integration documentation

### PR DESCRIPTION
**Pull request by enolfc**
# Summary

---

**Related issue :**

### CODE REVIEW

**Review by gwarf — APPROVED** [review]
LGTM

**Comment by enolfc** [comment]
> LGTM

You are fast... I plan to add some more stuff here :)

**Comment by enolfc** [comment]
@gwarf  can you take another look?

File: content/en/providers/cloud-compute/openstack/_index.md [hunk 1]
```diff
@@ -800,8 +799,7 @@ Apache server with [gridsite](https://github.com/CESNET/gridsite) and SSL
 support. GridSite is a set of extensions to the Apache 2.x webserver, which
 support Grid security based on X.509 certificates.
 
-Packages for gridsite can be obtained from CMD-OS-1. Follow the
-[CMD-OS-1 guidelines for getting the packages for your distribution](http://repository.egi.eu/category/os-distribution/cmd-os-1/).
+Packages for gridsite can be obtained from [CMD](https://repository.egi.eu/sw/production/cmd-os/1/).
```
Comment (gwarf) [inline content/en/providers/cloud-compute/openstack/_index.md:802]:
```suggestion
Packages for gridsite can be obtained from [CMD repository](https://repository.egi.eu/sw/production/cmd-os/1/).
```

File: content/en/providers/cloud-compute/openstack/_index.md [hunk 2]
```diff
@@ -1178,6 +1176,23 @@ have a topic like: `SITE_IFCA-LCG2_ENDPOINT_7513G0`.
...
+
+- `/etc/cloud-info-provider/openstack.rc` with the configuration of the
+  account used to log into your OpenStack and the location of the host
+  certificate that will be used to authenticate to the AMS.
+
+- `/etc/cloud-info-provider/openstack.yaml` with the cloud-info-provider
+  configuration. You need to enter the details about the VOs/projects that
+  the site is supporting.
+
+The appliance has a cron job that will connect to the configured OpenStack
```
Comment (gwarf) [inline content/en/providers/cloud-compute/openstack/_index.md:1193]:
```suggestion
The appliance has a cron job that will connect to the configured OpenStack API
```

**Review by gwarf — CHANGES_REQUESTED** [review]
Thanks!
Added a few suggestions, please check if they make sense.

**Review by gwarf — APPROVED** [review]
LGTM

**Pull request merged**